lib/cc/engine/rubymotion.rb

```diff
@@ -42,6 +40,29 @@ def run
 
       private
 
+      def files_to_analyze
```

Comment:
What do you think of pulling these methods out into their own class? Something that could call like `AnalyzableFiles.new(@engine_config).files`. I think that would keep this `Rubymotion` class a bit simpler and closer to following SRP